### PR TITLE
Fix formatting the internal workspace

### PR DESCRIPTION
**Pull request by bjorn3**

### CODE REVIEW

File: xtask/src/main.rs [hunk 1]
```diff
@@ -51,10 +39,28 @@ fn run_action(workspace: Workspace, target: &str, extra_args: &[String]) {
...
+        cmd.current_dir(ws_dir);
+
         cmd
     } else {
         let mut cmd = Command::new(env::var("CARGO").unwrap());
         cmd.arg(target);
+
+        let this_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
+        let ws_dir = match workspace {
+            Workspace::Internal => [&this_dir, "..", "internal_ws", "yktrace"]
```
Comment (vext01) [inline xtask/src/main.rs:57]: Why "yktrace" here?

Comment (bjorn3) [reply xtask/src/main.rs]: Should have been "ykshim". Fixed.

**Comment by vext01** [comment]
I assume you tested it, but mis-formatting something in the internal ws and seeing if `xtask fmt` catches it?

If so, please squash.

**Comment by bjorn3** [comment]
I have tested it.

**Comment by vext01** [comment]
Thanks for this!

**Pull request merged**